src/main/java/net/fabricmc/loom/extension/LoomFilesImpl.java

```diff
@@ -96,21 +94,6 @@ public boolean hasCustomNatives() {
 		return project.getProperties().get("fabric.loom.natives.dir") != null;
 	}
 
```

Comment:
Since you removed the `getNativeDirectory`, I guess should `hasCustomNatives` should also remove?

Reply:
That's still used in MinecraftProviderImpl, could possibly move it there as well